apertium-uzb-kaa.uzb-kaa.dix

```diff
@@ -19553,6 +19553,5526 @@
+<e>       <p><l>Daudova<s n="np"/><s n="cog"/></l>    <r>Daudova<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Dautovskiy<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Dautovskiy<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Davidchenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Davidchenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Davidov<s n="np"/><s n="cog"/></l>    <r>Davidov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Davidova<s n="np"/><s n="cog"/></l>    <r>Davidova<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Davidovich<s n="np"/><s n="pat"/></l>    <r>Davidovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Davidson<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Davidson<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Davinich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Davinich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Davison<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Davison<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Davivich<s n="np"/><s n="pat"/></l>    <r>Davivich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Davson<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Davson<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Davudovich<s n="np"/><s n="pat"/></l>    <r>Davudovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Davutbashich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Davutbashich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Davutbasich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Davutbasich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Davutoʻgʻli<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Davutóǵli<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Davutovich<s n="np"/><s n="pat"/></l>    <r>Davutovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Dayrabekov<s n="np"/><s n="cog"/></l>    <r>Dayrabekov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Dayrabekova<s n="np"/><s n="cog"/></l>    <r>Dayrabekova<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Dedkov<s n="np"/><s n="cog"/></l>    <r>Dedkov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Dedkova<s n="np"/><s n="cog"/></l>    <r>Dedkova<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Dedovich<s n="np"/><s n="pat"/></l>    <r>Dedovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Degtarenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Degtarenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>de<b/>Hoop<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>de<b/>Hoop<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Dejanovich<s n="np"/><s n="pat"/></l>    <r>Dejanovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Dekabristov<s n="np"/><s n="cog"/></l>    <r>Dekabristov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Dekabristova<s n="np"/><s n="cog"/></l>    <r>Dekabristova<s n="np"/><s n="cog"/></r></p></e>
```

Comment:
You should remove the feminine form of all the pairs, since the lemma is the masculine form, and the gender tag will transfer.